fiftyone/core/stages.py

```diff
@@ -5355,6 +5356,11 @@ def __init__(self, tags, bool=None, all=False):
             tags = [tags]
         else:
             tags = list(tags)
+            if not builtins.all([etau.is_str(t) for t in tags]):
```

Comment:
you should not have to import `builtins` like this. See `SortBy.validate` for a similar validation. Unless the `all` property on this class is causing issues?
also minor but all() takes any iterator and so does not need to be a list comprehension
```suggestion
            if not all(etau.is_str(t) for t in tags):
```

Reply:
The problem is that the MatchTags has an `all` kwarg that overwrites the builtin. The fact that it can take an iterator is good to know though!

Reply:
ah ok didnt see that kwarg. minor suggestion then
```suggestion
            if not builtins.all(etau.is_str(t) for t in tags):
```

Reply:
I like it! Thanks Stuart